jobs/build/ocp4/build.groovy

```diff
@@ -347,6 +348,12 @@ def stageBuildImages() {
         def r = buildlib.determine_build_failure_ratio(recordLog)
         if (r.total > 10 && r.ratio > 0.25 || r.total > 1 && r.failed == r.total) {
             echo "${r.failed} of ${r.total} image builds failed; probably not the owners' fault, will not spam"
+            if (r.failed == r.total) {
```

Comment:
there's one case when this won't catch a total failure; when there's only one image building.

so i'd move this outside the `if` above to catch that case too.